lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -95,17 +146,126 @@ public function get_item( $request ) {
+	*
+	* @since 6.5.0
+	*
+	* @param WP_Font_Collection $collection Collection object.
+	* @param WP_REST_Request    $request    Request object.
+	* @return array|WP_Error
+	*/
+	public function prepare_item_for_response( $collection, $request ) {
+		$fields = $this->get_fields_for_response( $request );
+		$item   = array();
+
+		$config_fields = array( 'slug', 'name', 'description' );
+		foreach ( $config_fields as $field ) {
+			if ( in_array( $field, $fields, true ) ) {
+				$item[ $field ] = $collection->$field;
+			}
+		}
+
+		$data_fields = array( 'font_families', 'categories' );
+		if ( in_array( 'font_families', $fields, true ) || in_array( 'categories', $fields, true ) ) {
+			$content = $collection->get_content();
+
+			// If there was an error getting the collection data, return the error.
+			if ( is_wp_error( $content ) ) {
+				$content->add_data( array( 'status' => 500 ) );
+				return $content;
+			}
+
+			foreach ( $data_fields as $field ) {
+				if ( in_array( $field, $fields, true ) ) {
+					$item[ $field ] = $content[ $field ];
+				}
+			}
+		}
+
+		return rest_ensure_response( $item );
+	}
+
+	/**
+	 * Retrieves the font collection's schema, conforming to JSON Schema.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'font-collection',
+			'type'       => 'object',
+			'properties' => array(
+				'slug'          => array(
+					'description' => __( 'Unique identifier for the font collection.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'edit' ),
+					'readonly'    => true,
+				),
+				'name'          => array(
+					'description' => __( 'The name for the font collection.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'edit' ),
+				),
+				'description'   => array(
+					'description' => __( 'The description for the font collection.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'edit' ),
+				),
+				'font_families' => array(
+					'description' => __( 'The font families for the font collection.', 'gutenberg' ),
+					'type'        => 'array',
+					'context'     => array( 'edit' ),
+				),
+				'categories'    => array(
+					'description' => __( 'The categories for the font collection.', 'gutenberg' ),
+					'type'        => 'array',
+					'context'     => array( 'edit' ),
+				),
+			),
+		);
+
+		$this->schema = $schema;
+
+		return $this->add_additional_fields_schema( $this->schema );
+	}
+
+	/**
+	 * Retrieves the search params for the font collections.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Collection parameters.
+	 */
+	public function get_collection_params() {
+		$query_params = parent::get_collection_params();
+
+		unset( $query_params['search'] );
+
```

Comment:
Add the context param to collection params.
```suggestion
		$query_params['context') = array(
			'description'       => __( 'Scope under which the request is made; determines fields present in response.' ),
			'type'              => 'string',
			'default'          => 'edit',
			'enum'            => array( 'edit' ),
			'sanitize_callback' => 'sanitize_key',
			'validate_callback' => 'rest_validate_request_arg',
		);

```

Reply:
Note, updated the above to use the existing controller method.

Reply:
added here: https://github.com/WordPress/gutenberg/pull/58222/commits/e545be01d31aab9f05e7f38fbc0a13720ddb7b22